Jenkinsfile

```
@@ -28,6 +29,11 @@ pipeline {
                 container('kaniko') {
                     sh "executor -c . --skip-tls-verify --digest-file image -d $HARBOR_HOST/rode-demo/rode-demo-node-app:${tag}"
                 }
+                container('curl-jq') {
+                    script {
+                        accessToken = sh(script: "./get-access-token.sh > access-token && cat access-token", returnStdout: true).trim()
```

Doesn't look like `accessToken` is used as a variable at all

```suggestion
                        sh "./get-access-token.sh > access-token"
```